pkg/api/mlflow/routes.go

```diff
@@ -31,38 +79,38 @@ func (r Router) Init(server fiber.Router) {
 	for _, prefix := range r.prefixList {
 		mainGroup := server.Group(prefix)
 
-		artifacts := mainGroup.Group("/artifacts")
-		artifacts.Get("/list", r.controller.ListArtifacts)
-
-		experiments := mainGroup.Group("/experiments")
-		experiments.Post("/create", r.controller.CreateExperiment)
-		experiments.Post("/delete", r.controller.DeleteExperiment)
-		experiments.Get("/get", r.controller.GetExperiment)
-		experiments.Get("/get-by-name", r.controller.GetExperimentByName)
-		experiments.Get("/list", r.controller.SearchExperiments)
-		experiments.Post("/restore", r.controller.RestoreExperiment)
-		experiments.Get("/search", r.controller.SearchExperiments)
-		experiments.Post("/search", r.controller.SearchExperiments)
-		experiments.Post("/set-experiment-tag", r.controller.SetExperimentTag)
-		experiments.Post("/update", r.controller.UpdateExperiment)
-
-		metrics := mainGroup.Group("/metrics")
-		metrics.Get("/get-history", r.controller.GetMetricHistory)
-		metrics.Get("/get-history-bulk", r.controller.GetMetricHistoryBulk)
-		metrics.Post("/get-histories", metric.GetMetricHistories)
-
-		runs := mainGroup.Group("/runs")
-		runs.Post("/create", r.controller.CreateRun)
-		runs.Post("/delete", r.controller.DeleteRun)
-		runs.Post("/delete-tag", r.controller.DeleteRunTag)
-		runs.Get("/get", r.controller.GetRun)
-		runs.Post("/log-batch", r.controller.LogBatch)
-		runs.Post("/log-metric", r.controller.LogMetric)
-		runs.Post("/log-parameter", r.controller.LogParam)
-		runs.Post("/restore", r.controller.RestoreRun)
-		runs.Post("/search", r.controller.SearchRuns)
-		runs.Post("/set-tag", r.controller.SetRunTag)
-		runs.Post("/update", r.controller.UpdateRun)
+		artifacts := mainGroup.Group(ArtifactsRoutePrefix)
```

Comment:
I have to say, I do prefer seeing the path here instead of constant (which is indirect)

Reply:
I don't know, maybe I have to revert this changes back.